server/src/http/background_send_metrics.rs

```diff
@@ -123,7 +123,7 @@ pub async fn send_metrics_task(
                     } else {
                         feature_refresher
                             .unleash_client
-                            .send_batch_metrics(batch.clone())
+                            .send_batch_metrics(batch.clone(), Some(interval.num_milliseconds()))
```

Comment:
even though Edge operates on seconds we send ms on the wire as the lowest common denominator between SDKs